### DISCUSSION TITLE
to many installations

### DISCUSSION THREAD

**Discussion by can-tls** [solidusio/solidus#4627] [Troubleshooting] [answered]
Hello folks,
I'm very new to solidus and did something wrong,
solidus wasnt working for me anymore because vips were missing, and what did I do? bunch of random things, I stashed the changes, made a new branch and installed it TWO more times, but nothing changes ofcourse. I eventually switched back to the first branch and figured out that I only needed to install vips and I got the default shop running but the problem now is that the categories and all the products are shown/loaded 3 times, I also ran rails db:migrate at some point but i dont think that that caused this, I think i destroyed something else. I know this isnt much information but any ideas?

**Comment by kennyadsl** [comment, marked as answer]
Hey @can-tls, are you talking about categories and products coming from Solidus seeds or were they added manually by yourself?

**Reply by can-tls** [reply]
Hey Kennyadsl! seeds from solidus yes, the default categories and products show up multiple times, for example the categories show "clothing caps bags mugs clothing caps bags mugs clothing caps bags mugs" and the shirts, hoodies etc also appear 3 times in the same way.

**Reply by waiting-for-dev** [reply]
As you run the installation multiple times, the seeds were loaded as many times. If you only have development data, you can run `bin/rails db:reset`

**Reply by kennyadsl** [reply]
Got it! I think you can fix it by resetting the database and seed data. You can try with:

```
bin/rails db:drop
bin/rails db:migrate
bin/rails db:seed
bin/rails spree_sample:load
```

Please note that this will delete all the currenlty existing data in the database, before re-entering the sample records. Let me know if that works!

**Reply by can-tls** [reply]
resetting the database and then reloading the spree samples fixed it, thank you guys! you made my day :D hope you have a wonderful day <3
